### ISSUE TITLE
angular-animate 1.3.0 conflict 

### ISSUE THREAD

**Issue by linslin** [2fdevs/bower-videogular#4]
Error is coming up if I try to use angular-videogular 0.7.2 with angular-animate 1.3.0

`Uncaught Error: [$injector:unpr] Unknown provider: $$qProvider <- $$q <- $animate <- $compile`

http://errors.angularjs.org/1.2.28/$injector/unpr?p0=%24%24qProvider%20%3C-%20%24%24q%20%3C-%20%24animate%20%3C-%20%24compile

Removing angular-videogular will fix that again. According to this link, its not true that angular-animate is wrong spelled or unknown. Dont know, maybe its a problem of angular-videogular. 

Here is my bower.json  

```
{
  "name": "test-app",
  "version": "0.0.0",
  "dependencies": {
    "angular": "^1.3.7",
    "json3": "^3.3.0",
    "es5-shim": "^4.0.0",
    "bootstrap-sass-official": "^3.2.0",
    "angular-animate": "^1.3.9",
    "angular-cookies": "^1.3.0",
    "angular-messages": "^1.3.0",
    "angular-resource": "^1.3.0",
    "angular-route": "^1.3.0",
    "angular-sanitize": "^1.3.0",
    "angular-touch": "^1.3.0",
    "angular-translate": "~2.4.2",
    "angular-translate-loader-static-files": "~2.4.2",
    "angular-rest-orm": "~0.4.5",
    "iScroll": "https://github.com/cubiq/iscroll.git",
    "ng-lodash": "~0.1.0",
    "local-storage-db": "git://github.com/knadh/localStorageDB.git",
    "pdf-js": "https://github.com/mozilla/pdf.js.git",
    "videogular": "0.7.2",
    "videogular-themes-default": "0.7.2"
  },
  "devDependencies": {
    "angular-mocks": "~1.3.0",
    "angular-scenario": "~1.3.0",
    "angular-animate": "~1.3.7",
    "animate.css": "~3.2.0"
  },
  "appPath": "app",
  "resolutions": {
    "angular": ">=1.2",
    "angular-sanitize": "^1.3.0"
  }
}

```

**Comment by 2fdevs** [comment]
You're using angular-animate 1.3.9 with angular 1.3.0, maybe that's the problem.

**Comment by linslin** [comment]
Nope. :( as i said -> it only happends, if I add "videogular" into bower. Using Animate 1.3.0 also throws the same error.

```
{
  "name": "test-app",
  "version": "0.0.0",
  "dependencies": {
    "angular": "^1.3.7",
    "json3": "^3.3.0",
    "es5-shim": "^4.0.0",
    "bootstrap-sass-official": "^3.2.0",
    "angular-animate": "^1.3.0",
    "angular-cookies": "^1.3.0",
    "angular-messages": "^1.3.0",
    "angular-resource": "^1.3.0",
    "angular-route": "^1.3.0",
    "angular-sanitize": "^1.3.0",
    "angular-touch": "^1.3.0",
    "angular-translate": "~2.4.2",
    "angular-translate-loader-static-files": "~2.4.2",
    "angular-rest-orm": "~0.4.5",
    "iScroll": "https://github.com/cubiq/iscroll.git",
    "ng-lodash": "~0.1.0",
    "local-storage-db": "git://github.com/knadh/localStorageDB.git",
    "pdf-js": "https://github.com/mozilla/pdf.js.git",
    "videogular": "0.7.2",
    "videogular-themes-default": "0.7.2"
  },
  "devDependencies": {
    "angular-mocks": "~1.3.0",
    "angular-scenario": "~1.3.0",
    "angular-animate": "~1.3.7",
    "animate.css": "~3.2.0"
  },
  "appPath": "app",
  "resolutions": {
    "angular": ">=1.2",
    "angular-sanitize": "^1.3.0"
  }
}
```

**Comment by 2fdevs** [comment]
I will give it a try with my dev version of Videogular and I will tell you if this is a videogular issue.

Thanks for testing anyway :)

**Comment by linslin** [comment]
Thanks for your response.

**Comment by linslin** [comment]
Whats the status here?

**Comment by 2fdevs** [comment]
Ok, it seems that Videogular 0.7.2 is changing your angular 1.3.11 version to angular 1.2.28 (as far as I tested in my machine).

I've solved this by forcing to install angular 1.3.11 with bower install angular#1.3.11 and selecting the option 4:

Unable to find a suitable version for angular, please choose one:
    1) angular#~1.2.0 which resolved to 1.2.28 and is required by videogular#0.7.2 
    2) angular#^1.3.7 which resolved to 1.3.11 and is required by test-app 
    3) angular#>=1.2 which resolved to 1.3.11 and is required by ng-lodash#0.1.0 
    4) angular#1.3.11 which resolved to 1.3.11Prefix the choice with ! to persist it to bower.json

? Answer:: 4
bower install       angular#1.3.11

That's my solution for now, I will update Videogular 1.0.0 to resolve to the newer versions of Angular, but I don't want to modify the current tags because that could be a problem for other people.

**Comment by linslin** [comment]
Thanks m8. I will try this soon.
